presto-hive/src/test/java/io/prestosql/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -876,12 +877,10 @@ public void testDecimalBackedByINT64()
 
     @Test
     public void testParquetShortDecimalWriteToPrestoDecimalWithNonMatchingScale()
+            throws Exception
     {
-        assertThatThrownBy(() -> {
-            MessageType parquetSchema = parseMessageType(format("message hive_decimal { optional INT64 test (DECIMAL(%d, %d)); }", 10, 1));
-            tester.testRoundTrip(javaLongObjectInspector, ImmutableList.of(1L), ImmutableList.of(1L), createDecimalType(10, 2), Optional.of(parquetSchema));
-        }).hasMessage("Presto decimal column type has different scale (2) than Parquet decimal column (1)")
```

Comment:
the change here is more that removing `assertThatThrownBy`. Why?

Reply:
Because now values are scaled (in the test the readValues were dummy) so we are actually testing a round-trip of read-write valeus